java/gradle.java/src/org/netbeans/modules/gradle/java/queries/TextDependencyScanner.java

```diff
@@ -0,0 +1,708 @@
+/*
```

Comment:
OMG. Just wanted to add an ANTLR4 Lexer and Parser for simple Groovy based Gradle projects. Anyway, I still have to do that, so this is welcome!

Reply:
Hm, I am working on output artifacts support and for that I'd need some other scanner, at least for simple name=value assignments to various task properties. I plan to supply the list of _possible_ property names (and structure members) from introspecting the tasks / extensions in the project, so the scanner could better ignore 'the unknown' stuff without having to be a full parser.
Stay tuned, it's still in very rough state